src/TodoistApi.ts

```diff
@@ -55,6 +54,10 @@ import {
     validateUserArray,
 } from './utils/validators'
 
+function generatePath(...segments: string[]): string {
+    return segments.join('/')
```

Comment:
This feels very fragile as we could easily have situations with `//` and broken paths 👀

Reply:
Maybe we could add a code comment to make the desired format obvious?

Reply:
Added a comment in c4e3f61d3943646556f851c01d01a2b029669a03.

In future refactoring, it is probably best to use literals directly, like this:

```
`tasks/${id}`
```

This is both easier to follow and to validate using literal-types.